### ISSUE TITLE
Issues with automatic provisioning of scenarios with Fleet Server and Agent

### ISSUE THREAD

**Issue by jsoriano** [elastic/elastic-agent#2231]

**Comment by jsoriano** [comment]
> @jsoriano we're not sure if this is needed in the default instance, but we were having some issues with the docker setup yesterday and found that the package-registry instance needed this added to its docker config in-order to come online:
> 
> ```
> security_opt:
>   - seccomp:unconfined
> ```
> 
> Otherwise we received an immediate error:
> 
> runtime/cgo: pthread_create failed: Operation not permitted

I don't think this is related to this issue. I haven't seen this kind of problems with package-registry before. @joeperuzzi could you please create an issue in https://github.com/elastic/elastic-package repository with information about your environment and the version of elastic-package that you are using?

**Comment by chrispangg** [comment]
Both elastic-agent and elastic-agent-complete images are still having this issue. What is the workaround for this if I am using docker-compose to provision the fleet server?
